pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -158,15 +173,6 @@ def execute(
         circuits: QuantumTape_or_Batch,
         execution_config: ExecutionConfig = DefaultExecutionConfig,
     ):
-        """Execute a circuit or a batch of circuits and turn it into results.
```

Comment:
Why is the docstring removed?

Reply:
the parent class already provides the exact same docstring, only with more detail